docs/calamari/CommunityCollatorsProgram.md

```diff
@@ -0,0 +1,305 @@
+# 🧩  Community Collators Program
+
+## Requirements
+
+1. Hardware Requirement
+
+    a. If running on your own machine, hardware requirement, internet bandwidth (we prefer collators with private node for better decentralization).
+    
+    ```jsx
+    cpu:8 cores
+    memory: 32 GB of ram
+    disk space: 240 GB of disk space dedicated to the blockchain basepath
+    bandwidth: 100 Mbps
+    recommended region: EU or US
```

Comment:
I think @stechu mentioned we have candidates from Asia already, if we argue *for better decentralization* above, we shouldn't centralize the collator regions, i'd remove the `recommended region: EU or US` part